packages/react/src/components/Web3Button/index.tsx

```diff
@@ -122,13 +124,14 @@ export const Web3Button = <TAction extends ActionFn>({
     },
   );
   if (!address) {
-    return <ConnectWallet {...themeProps} />;
+    return <ConnectWallet className={className} {...themeProps} />;
```

Comment:
wondering if we could make the Web3Button work even without our standard provider. Could we skip this if we're not in our own ThirdwebProvider context ?